### PR TITLE
Himank/ar 1464 unit testing with ava

### PR DESCRIPTION
**Pull request by steel-feel**
Unit tests written in AVA

### CODE REVIEW

File: tests/test.ts [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+}
+
+
+function nockSetup()
+{   
+
+   
+   
+
+    nock('https://gateway02.arcana.network/').defaultReplyHeaders(nockOptions).persist()
```
Comment (SauravKanchan) [inline tests/test.ts:172]: use versioned APIs

https://gateway02.arcana.network/api/v1/

File: tests/test.ts [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+    .get("/get-nonce/")
+    .query(true)
+    .reply(200, "0")
+    .get("/get-address/")
+    .query(true)
+    .reply(200, { address: "0x98f92D5B2Eb666f993c5930624C2a73a3ED5B158" })
+    .intercept("/get-address/", "OPTIONS")
+    .query(true)
+    .reply(200, { address: "0x98f92D5B2Eb666f993c5930624C2a73a3ED5B158" },{'access-control-allow-headers': 'Authorization'})
+    .get("/api/get-address/")
```
Comment (SauravKanchan) [inline tests/test.ts:190]: /get-node-address/

File: tests/test.ts [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+    let after = await access.getSharedUsers(did);
+
+    t.truthy(tx);
+
+    t.is(before.includes(receiverWallet.address), true);
+    t.is(after.includes(receiverWallet.address), false);
+    t.is(before.length - after.length, 1);
+
+    await nock('https://gateway02.arcana.network/').defaultReplyHeaders(nockOptions)
+        .get("/api/shared-files/")
```
Comment (SauravKanchan) [inline tests/test.ts:331]: remove /api/ through out file

**Review by SauravKanchan — CHANGES_REQUESTED** [review]

File: test/test.js [hunk 4]
```diff
@@ -1,234 +0,0 @@
-const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```
Comment (SauravKanchan) [inline test/test.js:1]: Let the HTML version of testing exist for sometime

**Review by SauravKanchan — CHANGES_REQUESTED** [review]

File: test/test.js (hunk 4 above)
Comment (DudeWhoCode) [reply test/test.js:1]: If this is about old integration/browser test, don't remove it. Move it to `integrationtests` folder or something.

Comment (steel-feel) [reply test/test.js:1]: Created separate branch for the change AR-1880-integration-tests

**Comment by steel-feel** [comment]
API version changes done

**Pull request merged**